<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-UB7KQ0Q" failures="1" tests="8" name="testcases.LoginTestProctor" time="77.192" errors="0" timestamp="2025-07-10T14:38:18 IST" skipped="0">
  <testcase classname="testcases.LoginTestProctor" name="verifyForgotPassword" time="9.230"/>
  <testcase classname="testcases.LoginTestProctor" name="verifyWithInvalidCredentials" time="2.471"/>
  <testcase classname="testcases.LoginTestProctor" name="verifyWithInvalidCredentials" time="2.492">
    <failure message="The following asserts failed:
	expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at testcases.LoginTestProctor.verifyWithInvalidCredentials(LoginTestProctor.java:63)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- verifyWithInvalidCredentials -->
  <testcase classname="testcases.LoginTestProctor" name="verifyWithInvalidCredentials" time="2.418"/>
  <testcase classname="testcases.LoginTestProctor" name="verifyWithInvalidCredentials" time="2.662"/>
  <testcase classname="testcases.LoginTestProctor" name="verifyWithValidCredentials" time="4.299"/>
  <testcase classname="testcases.LoginTestProctor" name="verifyResetPassword" time="36.980">
    <system-out>
      <![CDATA[Navigated to Reset Password page
Entered new password: daya@123
Entered confirm password: Daya@123
Success message not visible. Checking for error messages.]]>
    </system-out>
  </testcase> <!-- verifyResetPassword -->
  <testcase classname="testcases.LoginTestProctor" name="verifyResetPassword" time="16.640">
    <system-out>
      <![CDATA[Navigated to Reset Password page
Entered new password: Daya@123456
Entered confirm password: Daya@123456
Password changed successfully with: Daya@123456 / Daya@123456]]>
    </system-out>
  </testcase> <!-- verifyResetPassword -->
  <system-out>
    <![CDATA[Navigated to Reset Password page
Entered new password: daya@123
Entered confirm password: Daya@123
Success message not visible. Checking for error messages.
Navigated to Reset Password page
Entered new password: Daya@123456
Entered confirm password: Daya@123456
Password changed successfully with: Daya@123456 / Daya@123456]]>
  </system-out>
</testsuite> <!-- testcases.LoginTestProctor -->
